### ISSUE TITLE
Will rejected by Google

### ISSUE THREAD

**Issue by AppWerft** [equationl/Tiny-FFmpeg-Android#2]
In my app I downloaded the binary from internet for usage. This is forbidden by rule.

**Comment by equationl** [comment]
@AppWerft Yes, it may be banned by google play, but I don’t know why my app uses this library, but it can also be uploaded to google play.

**Comment by AppWerft** [comment]
You are a lucky man. I have built a Titanium module that loads binary from net and copy to app folder. This was banned 1 week ago. 
This is my patch:
https://github.com/AppWerft/Ti.FFmpeg/blob/master/android/src/com/github/hiteshsondhi88/libffmpeg/FileUtils.java#L28-L47

**Comment by equationl** [comment]
What does Google say?

**Comment by AppWerft** [comment]
It is not allowed to load executables like (jar, aar …) after installation. Now I think about zipping. This reduced about 50%

**Comment by equationl** [comment]
Ok, Get it.

**Comment by HBiSoft** [comment]
@equationl This library has 2 major issues, I decided to comment here instead of opening a new issue.

The first issue is that the files are located in the `assets` folder. Instead, the binaries should be renamed to `lib..ffmpeg..so` and placed in the jniLibs folder. If you are targeting 29, you can not access `/data/user/0/com.package.name/files/ffmpeg`.

Secondly, you are "modifying" code after installation, this is against Google's policy ([see changes in Android Q](https://developer.android.com/about/versions/10/behavior-changes-all#execute-permission)):

>Code must be loaded as-is, and must not be modified.

Doing what you are doing will most definitely get your application banned.

**Comment by equationl** [comment]
@HBiSoft Yes, I see. I am thinking how to solve it
